src/app/api/rooms/[roomId]/reviews/route.ts

```
@@ -10,14 +10,19 @@ import {
-import { Review, ReviewParams } from '@/types/review';
+import { ReviewParams, ReviewSummarize } from '@/types/review';
 
 export async function GET(
   request: NextRequest,
   { params }: { params: ReviewParams },
-): Promise<CustomResponse<PaginationResponse<Review> | undefined>> {
+): Promise<CustomResponse<PaginationResponse<ReviewSummarize> | undefined>> {
   try {
     const roomId = +params.roomId;
+
+    if (isNaN(roomId)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)